src/main/kotlin/org/rust/cargo/project/model/impl/TestCargoProjectsServiceImpl.kt

```diff
@@ -132,3 +145,9 @@ class TestCargoProjectsServiceImpl(project: Project) : CargoProjectsServiceImpl(
             ?: error("Timeout when refreshing a test Cargo project")
     }
 }
+
+val DEFAULT_EDITION_FOR_TESTS: Edition
```

Comment:
```suggestion
@get:TestOnly
val DEFAULT_EDITION_FOR_TESTS: Edition
```

It allows `JVM languages | Test-only usage in production code` inspection detects usage of this property in wrong place